### DISCUSSION TITLE
Plumber 2 - Multi site/language editor/publishers

### DISCUSSION THREAD

**Discussion by gurumurthyjv** [nathanwoulfe/Plumber-2#119] [General]
@nathanwoulfe ,

HI,

I was exploring Plumber 2 for my one of Umbraco v8.14, the solution seems to be great for managing multi level workflows.

I have an question - do this workflows can be managed for multi language site in v8 ? For example, I need editors and publishers  specific to variants or languages, is that possible from Plumber 2?

Like I have 2 variants/site one is uk and other is ar,.

Thanks,
Gurumurthy J V

**Comment by nathanwoulfe** [comment]
Hi @gurumurthyjv 

Workflows can most definitely be applied on multi-language sites, or will work equally well in a multi-site Umbraco install. For multi-language, each variant for a given node can have a unique workflow assigned, plus has different options when submitting changes for approval. 

For example, on single node:

en-UK variant has workflow GroupA => GroupB => GroupC
ar variant has workflow GroupD => GroupE => GroupC
fr variant inherits workflow from an ancestor node - GroupF => GroupG => GroupC

When submitting changes for approval, an editor can choose submit the current (active) variant, all variants via their assigned workflow in individual workflow processes, or all variants via the workflow assigned on the current (active) variant, in a single workflow process. 

Variant permissions follow the same inheritance pattern as invariant - on a given node, the explicitly assigned workflow permissions are used first, if none exist Plumber checks for permissions assigned to the content type, then if still none exist, traverses the content tree to find an ancestor node with explicit permissions. This means variants on the same node can follow different permissions models and be published independently or as a set. 

Long story short, yes, Plumber works nicely with multi language sites.

**Reply by gurumurthyjv** [reply]
Thanks @nathanwoulfe , for your response,
Will try to implement this.

Thanks,
Gurumurthy J V
